### PR TITLE
Make puller pause configurable

### PR DESCRIPTION
**Pull request by AudriusButkevicius**

### CODE REVIEW

File: lib/model/rwfolder.go [hunk 1]
```diff
@@ -143,6 +141,21 @@ func newRWFolder(m *Model, shortID uint64, cfg config.FolderConfiguration) *rwFo
 	if p.pullers == 0 {
 		p.pullers = defaultPullers
 	}
+	if p.sleep == 0 {
+		p.sleep = defaultPullerSleep
```
Comment (calmh) [inline lib/model/rwfolder.go]: This gets overwritten a few lines down

File: lib/model/rwfolder.go [hunk 2]
```diff
@@ -143,6 +141,21 @@ func newRWFolder(m *Model, shortID uint64, cfg config.FolderConfiguration) *rwFo
...
+	}
+
+	if cfg.PullerPauseS == 0 {
+		p.pause = defaultPullerPause
+	} else {
+		p.pause = time.Duration(cfg.PullerPauseS) * time.Second
+	}
+
+	if cfg.PullerSleepS == 0 {
+		p.sleep = defaultPullerPause
```
Comment (calmh) [inline lib/model/rwfolder.go]: `defaultPullerSleep` I think

File: lib/config/config.go [hunk 3]
```diff
@@ -109,6 +109,8 @@ type FolderConfiguration struct {
 	Order                 PullOrder                   `xml:"order" json:"order"`
 	IgnoreDelete          bool                        `xml:"ignoreDelete" json:"ignoreDelete"`
 	ScanProgressIntervalS int                         `xml:"scanProgressInterval" json:"scanProgressInterval"` // Set to a negative value to disable. Value of 0 will get replaced with value of 2 (default value)
+	PullerSleepS          int                         `xml:"pullerSleep" json:"pullerSleep"`
```
Comment (calmh) [inline lib/config/config.go]: with `...S` suffix to indicate seconds, on both of these, json+xml?

**Comment by AudriusButkevicius** [comment]
Fixed.
Writing code at 1AM on a work day is difficult.

**Comment by calmh** [comment]
:D

**Pull request merged**